chispa/default_formats.py

```diff
@@ -3,7 +3,7 @@
 
 @dataclass
 class DefaultFormats:
-    mismatched_rows = ["red"]
-    matched_rows = ["blue"]
+    mismatched_rows = ("red",)
+    matched_rows = ("blue",)
```

Comment:
While I'm OK with that change, it looks strange to me. Which Ruff rule triggered it?

Reply:
Thanks for checking, well spotted! I initially did that change because of `RUF012`(Mutable class attributes). But then I decided to add that to the `exclude`, since IMO this class requires refactoring or removal anyway. So we'll see how we deal with this later.